jupyterbook/_config.yml

```diff
@@ -4,7 +4,7 @@ author: IOOS # The author of the book
 logo: images/IOOS_logo.png # A path to the book logo
 
 # Patterns to skip when building the book. Can be glob-style (e.g. "*skip.ipynb")
-exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints"]
+exclude_patterns: [_build, Thumbs.db, .DS_Store, "**.ipynb_checkpoints", "**.archived.ipynb"]
```

Comment:
@MathewBiddle this should work. We only need to append the `.archived` and we can still view them on GH or nbviewer, but they won't get build as part of the book.

Reply:
Awesome! Thanks for looking into this.